tests/semantic/witness_check2/witness_check2.dl

```diff
@@ -0,0 +1,32 @@
+.decl b(x:number)
```

Comment:
Our usual copyright header should be at the start. After that, a brief comment about the point of the test would be good.